Step04/app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <group android:checkableBehavior="single">
+        <item
+            android:id="@+id/nav_camera"
+            android:icon="@drawable/ic_menu_camera"
+            android:title="Import" />
+    </group>
```

Comment:
@kwoolytech Oh are you planning to take the picture and import it into the app ? interesting :D